src/LiveComponent/doc/index.rst

```diff
@@ -352,6 +352,21 @@ clicked). To do that, use ``norender`` modifier:
 
     <input data-model="norender|max">
 
+For a form using ``ComponentWithFormTrait``, override the ``getDataModelValue()`` method::
+
+private function getDataModelValue()
+{
+    return 'norender|*';
+}
+
+.. tip::
+
+    You can also define this value inside Twig::
+
+.. code-block:: html+twig
```

Comment:
The code block should be indented so that it's inside of the `tip`. And then the actual code below should be indented 4 more.